docker-compose.yaml

```diff
@@ -1,27 +1,58 @@
-version: '3'
+version: '3.2'
 services:
-  django:
+  start_services:
+    image: jwilder/dockerize
+    command: "-wait tcp://db:5432 -timeout 30s"
+    depends_on:
+      - db
+  start_applications:
+    image: jwilder/dockerize
+    command: "-wait tcp://cla_frontend:80 -wait tcp://cla_frontend:8005 -wait tcp://cla_backend:80 -timeout 180s"
+    depends_on:
+      - cla_frontend
+      - cla_backend
+
+  # services
+  db:
+    image: postgres:9.6-alpine
```

Comment:
Production uses PostgreSQL 9.4.9. Not sure if we should pin to the same version here.

Reply:
👍 Will do